examples/hyperparameter/run_benchmark.py

```diff
@@ -0,0 +1,51 @@
+"""
+./scheduler 52.50.92.141:10001
+
+import orchpy.services as services
+test_path = "/home/ubuntu/orch/examples/hyperparameter/benchmark.py"
+services.start_node("52.50.92.141:10001", "52.50.92.141", 1, worker_path=test_path)
+
+import orchpy.services as services
+test_path = "/home/ubuntu/halo/examples/tensorflow/benchmark.py"
+services.start_node("52.50.92.141:10001", "52.49.170.133", 1, worker_path=test_path)
+
+import orchpy.services as services
+test_path = "/home/ubuntu/halo/examples/tensorflow/benchmark.py"
+services.start_node("52.50.92.141:10001", "52.51.101.205", 1, worker_path=test_path)
+"""
+
+import orchpy as op
+import orchpy.services as services
+import hyperparameter
+import os
+import time
+import numpy as np
+
+
+test_path = "/home/ubuntu/photon/examples/hyperparameter/benchmark.py"
```

Comment:
this path probably shouldn't be hardcoded because someone might install ray in a different directory (e.g., not in /home/ubuntu)